### PR TITLE
Tempest cleanups

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -2984,20 +2984,21 @@ function oncontroller_tempest_legacy()
...
         "prepare cnftools image"
+
     pushd /var/lib/openstack-tempest-test
-    echo 1 > /proc/sys/kernel/sysrq
-    if iscloudver 5plus; then
-        if tempest help cleanup; then
-            tempest cleanup --init-saved-state
-        else
-            /usr/bin/tempest-cleanup --init-saved-state || :
+        echo 1 > /proc/sys/kernel/sysrq
```
Comment (aplanas) [inline scripts/qa_crowbarsetup.sh]: this indentation is not used consistently in the rest of the code, maybe we can avoid doing this here and do that in a different PR that address other cases

**Comment by aplanas** [comment]
Except for the last commit: +1

**Comment by sayalilunkad** [comment]
+1
